### PR TITLE
Join with Backend, add Anti-Cheat Inits and fixes for initial

### PR DESCRIPTION
**Pull request by 0xk1f0**
#### Basic Requirements

- [x] I have added Documentation
- [x] I have tested my code
- [x] I have synced my branch with main

#### Describe your PR, what does it fix/add?

This joins the deployment backend with the frontend and adds Anti-Cheat bits to the mix, which is still pending. 

#### Is there anything you want to mention? (unchecked code, possible bugs, found problems, breaking compatibility, etc.)

Currently, what is missing is the double checkup with the deployment backend for the confirmation of currently running challenges. This is crucial because we use it to check what is currently running, avoid restarts, avoid spawning multiple instances per team and limit running instances to teams.

#### Is it ready for merging, or does it need work?

Basically what is mentioned above plus visual enhancements for the team page, which looks a bit naked as of now.

This also closes 3 issues.

### CODE REVIEW

File: src/components/Scoreboards.svelte [hunk 1]
```diff
@@ -185,48 +228,60 @@
...
-            sortedEvents.push({ value: element.id, name: element.event_name });
+        events.forEach((element) => {
+            sortedEvents.push({ value: element, name: element.event_name });
         });
     }
 
-    async function refreshEventScoring(eventID: string) {
-        const SOLVES = await requestWrapper(false, { type: 'event-solves', data: eventID });
-        events = (await SOLVES.json()).data;
+    async function refreshEventScoring(id: string) {
```
Comment (Merrixx) [inline src/components/Scoreboards.svelte:273]: Could remove the console.logs

File: src/lib/actions.ts [hunk 2]
```diff
@@ -285,68 +314,160 @@ class DatabaseActions {
...
+     */
+    async deployTeamChallenge(
+        genFlag: string,
+        team_id: string,
+        challenge_id: string,
+        event_id: string
+    ): Promise<boolean> {
         const RES = await DB_ADAPTER.select().from(challenges).where(eq(challenges.id, challenge_id));
         if (RES.length > 0) {
+            console.log(JSON.stringify({
```
Comment (Merrixx) [inline src/lib/actions.ts:356]: Remove here aswell if unnecessary

**Review by Merrixx — APPROVED** [review]
Approved 🤠

File: src/components/Scoreboards.svelte (hunk 1 above)
Comment (0xk1f0) [reply src/components/Scoreboards.svelte:272]: Done

File: src/lib/actions.ts (hunk 2 above)
Comment (0xk1f0) [reply src/lib/actions.ts]: Done

**Review by Merrixx — APPROVED** [review]
Checked (2) and approved 🤠
